### PR TITLE
Fix package name

### PR DESCRIPTION
**Pull request by steeef**
The package for `rpmbuild` was named incorrectly. Easy fix.

### CODE REVIEW

**Comment by steeef** [comment]
Reduced the amount of commands, and made it easier to switch versions. The spec file had `Version: 1.8` in it, so I added a sed command to replace with the one requested.

File: contrib/rpm/README.md [hunk 1]
```diff
@@ -3,21 +3,26 @@ Packaging Instructions
...
 
-    cd SOURCES
-    wget https://github.com/driskell/log-courier/archive/vVERSION.zip
-    cd SPECS
-    wget https://raw.githubusercontent.com/driskell/log-courier/vVERSION/contrib/rpm/log-courier.spec
-    rpmbuild -ba log-courier.spec
+```
+export VERSION=<your version here> # set to the version you wish to download and build
+wget -P rpmbuild/SOURCES https://github.com/driskell/log-courier/archive/v${VERSION}.zip
+wget -P rpmbuild/SOURCES https://raw.githubusercontent.com/driskell/log-courier/v${VERSION}/contrib/rpm/log-courier.spec
```
Comment (driskell) [inline contrib/rpm/README.md]: Did you mean SPECS here?

**Comment by driskell** [comment]
Thanks looks good! Guess I wrote this from memory and made a mistake or too. Sorry!

**Comment by steeef** [comment]
I did mean SPECS, thanks for catching that. Fixed!

**Pull request merged**